docs/content/documentation/deployment/github-pages.md

```diff
@@ -33,19 +33,19 @@ on: push
 name: Build and deploy GH Pages
 jobs:
   build:
-    name: shalzz/zola-deploy-action
+    name: Deploy
     runs-on: ubuntu-latest
     steps:
-    # Checkout
-    - uses: actions/checkout@master
-    # Build & deploy
-    - name: shalzz/zola-deploy-action
-      uses: shalzz/zola-deploy-action@v0.12.0
-      env:
-        # Target branch
-        PAGES_BRANCH: gh-pages
-        # Provide personal access token
-        TOKEN: ${{ secrets.TOKEN }}
+      # Checkout
+      - uses: actions/checkout@v2
+      # Build & deploy
+      - name: Deploy to gh-pages branch
+        uses: shalzz/zola-deploy-action@v0.12.0
```

Comment:
let's put 0.13 here as well

Reply:
Done in abf6ec71e2518230281bb478e591d5f75e368032